lib/anycable/rails/config.rb

```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
+#
+
+require 'anycable/config'
```

Comment:
Could you please RuboCop offense for this line (double qoutes required)?
```suggestion
require "anycable/config"
```